gen/pb-ruby/README.md

```diff
@@ -0,0 +1,48 @@
+# sigstore_protobuf_specs
+
+These are the Ruby language bindings for Sigstore's protobuf specs.
+
+See the [repository's README](https://github.com/sigstore/protobuf-specs)
+for more information.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'sigstore_protobuf_specs'
+```
+
+And then execute:
+```bash
+    $ bundle install
+```
+Or install it yourself as:
+```bash
+    $ gem install sigstore_protobuf_specs
+```
+
+## Usage
+
+Import all the bindings:
+
+```ruby
+require 'sigstore_protobuf_specs'
+```
+
+Or you can import them individually:
+
+```ruby
+require 'sigstore_bundle_pb'
+```
+
+See what is available in `gen/pb-ruby/lib/`.
+
+## Releasing
+
+Make sure you update the version in `gen/pb-ruby/lib/sigstore_protobuf_specs/version.rb`
```

Comment:
I wonder if you should add some information on the ownership of the gem. And note that this repository is equipped to publish.